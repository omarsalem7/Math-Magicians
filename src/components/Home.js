import React from 'react';

const Home = () => (
  <div className="home">
    <h2>Welcome to our page</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
      tempor incididunt ut labore et dolore magna aliqua. Bibendum neque egestas
      congue quisque egestas diam in arcu cursus. Non nisi est sit amet
      facilisis magna. Amet tellus cras adipiscing enim eu turpis egestas.
      Nullam vehicula ipsum a arcu cursus vitae congue mauris. Et malesuada
      fames ac turpis egestas sed. Elit ullamcorper dignissim cras tincidunt
      lobortis
    </p>
  </div>
);

export default Home;
