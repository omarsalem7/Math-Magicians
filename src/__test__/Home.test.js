import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('test home component', () => {
  it('Home renders correct', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
