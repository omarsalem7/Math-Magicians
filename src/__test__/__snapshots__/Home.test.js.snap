// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing home component Home renders correctly 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum neque egestas congue quisque egestas diam in arcu cursus. Non nisi est sit amet facilisis magna. Amet tellus cras adipiscing enim eu turpis egestas. Nullam vehicula ipsum a arcu cursus vitae congue mauris. Et malesuada fames ac turpis egestas sed. Elit ullamcorper dignissim cras tincidunt lobortis
  </p>
</div>
`;

exports[`test home component Home renders correct 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum neque egestas congue quisque egestas diam in arcu cursus. Non nisi est sit amet facilisis magna. Amet tellus cras adipiscing enim eu turpis egestas. Nullam vehicula ipsum a arcu cursus vitae congue mauris. Et malesuada fames ac turpis egestas sed. Elit ullamcorper dignissim cras tincidunt lobortis
  </p>
</div>
`;

exports[`testg home component Home renders correctly 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum neque egestas congue quisque egestas diam in arcu cursus. Non nisi est sit amet facilisis magna. Amet tellus cras adipiscing enim eu turpis egestas. Nullam vehicula ipsum a arcu cursus vitae congue mauris. Et malesuada fames ac turpis egestas sed. Elit ullamcorper dignissim cras tincidunt lobortis
  </p>
</div>
`;
